openpype/tools/loader/widgets.py

```diff
@@ -741,6 +741,16 @@ def set_version(self, version_doc=None, version_id=None):
             "source": source_label
         }
 
+        # Add colorspace info to Version Info tab
+        version_colorspace = version_doc["data"].get("colorspace")
+        colorspace_html = ""
```

Comment:
Isn't the colorspace on the representations instead of the version?

Reply:
I checked a few entries in MongoDB and I can only see it on the version data

Reply:
In fact @BigRoy is right @fabiaserra. The new colorspaceData distribution is kept on representation level. The version data colorspace key is still available but will be removed soon.

Reply:
yeah fair enough, no big deal, it was just a fast utility for me while testing the OpenRV integration to make sure what I was loading was getting the expected colorspace, happy to close it